deploy/build-agent/build-push-docker-hub.sh

```diff
@@ -9,5 +9,5 @@ if [ -n "$GIT_STATUS_OUTPUT" ]; then
 fi
 
 GIT_COMMIT_SHORT=$(git log -1 --format=%h)
-DOCKER_BUILDKIT=1 docker build -t stamtc/devops-build-agent:latest -t stamtc/devops-build-agent:$GIT_COMMIT_SHORT --platform linux/amd64 .
```

Comment:
I take it that DOCKER_BUILDKIT=1 is no longer needed.  Can you explain why please?  Did the standard docker build task get better?

Reply:
https://docs.docker.com/build/buildkit/#:~:text=Getting%20started,t%20need%20to%20enable%20BuildKit.

Pleased to report it's now a default...

`BuildKit is the default builder for users on Docker Desktop, and Docker Engine as of version 23.0.`